src/Ocelot/Requester/MessageInvokerPool.cs

```diff
@@ -31,17 +38,15 @@ public HttpMessageInvoker Get(DownstreamRoute downstreamRoute)
     }
 
     public void Clear() => _handlersPool.Clear();
-
-    /// <summary>
-    /// TODO This should be configurable and available as global config parameter in ocelot.json.
-    /// </summary>
-    public const int DefaultRequestTimeoutSeconds = 90;
+    
+    private readonly int _defaultRequestTimeoutSeconds;
```

Comment:
Seems the member should be converted to public property

Reply:
fixed